x/gov/module.go

```diff
@@ -186,9 +188,9 @@ type GovOutputs struct {
 }
 
 func ProvideModule(in GovInputs) GovOutputs {
-	kConfig := govtypes.DefaultConfig()
+	Config := govtypes.DefaultConfig()
```

Comment:
```suggestion
	defaultConfig := govtypes.DefaultConfig()
```

a bit weird to have it cased `Config`.